dali/operators/audio/mel_scale/mel_filter_bank.cc

```diff
@@ -20,39 +20,51 @@
 namespace dali {
 
 DALI_SCHEMA(MelFilterBank)
-    .DocStr(R"code(Converts a Spectrogram to a mel Spectrogram using triangular filter banks.
-Expects an input with 2 or more dimensions where the last two dimensions correspond to the
-fft bin index and the window index respectively.)code")
+    .DocStr(R"code(Converts a Spectrogram to a mel spectrogram by using triangular filter banks.
```

Comment:
```suggestion
    .DocStr(R"code(Converts a spectrogram to a mel spectrogram by using triangular filter banks.
```
It converts a spectrogram (data), not the operator.

Reply:
Done